packages/cdn/src/index.ts

```diff
@@ -7,8 +7,8 @@ app.use(cors());
 
 const sourceCache = new Map<string, Uint8Array>();
 
-const query = async (method: string, accountId: string, filename: string, part?: number) => {
-  const provider = new JsonRpcProvider({ url: 'https://rpc.testnet.near.org' });
+const query = async (method: string, accountId: string, filename: string, part?: number, application?: string) => {
```

Comment:
Looks like the list of arguments is growing, I think it might make sense to avoid [connascence of position](https://connascence.io/position.html) by creating an object param as a config of the query. How does the similar in `provider.query` call look to you?

Reply:
Yup this whole thing needs a rework, it's really hacky.